src/main/java/com/atlassian/jira/cloud/jenkins/deploymentinfo/service/JiraDeploymentInfoSenderImpl.java

```diff
@@ -68,40 +69,42 @@ public JiraSendInfoResponse sendDeploymentInfo(final JiraDeploymentInfoRequest r
             return JiraCommonResponse.failureSiteConfigNotFound(jiraSite);
         }
 
+        final String resolvedSiteConfig = maybeSiteConfig.get().getSite();
```

Comment:
/nit you can move final String resolvedSiteConfig = maybeSiteConfig.get().getSite(); after final JiraCloudSiteConfig siteConfig = maybeSiteConfig.get();